data/data/coreos/rhcos.json

```diff
@@ -350,6 +350,12 @@
               "bucket": "rhcos-powervs-images-eu-de",
               "url": "https://s3.eu-de.cloud-object-storage.appdomain.cloud/rhcos-powervs-images-eu-de/rhcos-415-92-202310310037-0-ppc64le-powervs.ova.gz"
             },
+            "eu-es": {
```

Comment:
Aren't these auto generated? Ex: https://github.com/openshift/installer/pull/7654

Reply:
I believe it is but we don't want to wait for it to get generated to add the region

Reply:
Pointing to a different region sounds hacky.

Reply:
There is a PR up to update the json: https://github.com/openshift/installer/pull/7770

Reply:
awesome! I'll remove it from this PR